Confus/Confus/Flag.cpp

```diff
@@ -13,15 +13,15 @@
 
 namespace Confus {
 
-	Flag::Flag(irr::IrrlichtDevice* a_Device, ETeamIdentifier a_TeamIdentifier) : m_TeamIdentifier(new ETeamIdentifier(a_TeamIdentifier)),
-				m_FlagStatus(new EFlagEnum(EFlagEnum::FlagBase)) {
+	Flag::Flag(irr::IrrlichtDevice* a_Device, ETeamIdentifier a_TeamIdentifier) : m_TeamIdentifier(a_TeamIdentifier),
+				m_FlagStatus(EFlagEnum::FlagBase) {
         //Get drivers to load model
         auto sceneManager = a_Device->getSceneManager();
         auto videoDriver = a_Device->getVideoDriver();
 
 		//Flags
-		m_StartPosition = new irr::core::vector3df();
-		m_StartRotation = new irr::core::vector3df();
+		m_StartPosition;
+		m_StartRotation;
```

Comment:
Ì think its still a good idea to set these to irr::core::vector3df as a default. You might end up with 3 uninitialized floats otherwise